### DISCUSSION TITLE
Has anybody got rclone_rd working on unraid

### DISCUSSION THREAD

**Discussion by mash2k3** [itsToggle/plex_debrid#300] [General]
Has anybody got rclone_rd working on unraid?
I tried to pass the commands via post arguments but having no success.

**Comment by mash2k3** [comment]
Solved

**Reply by molesza** [reply]
Glad to hear its possible! Can you direct the rest of us to the solution please?

**Reply by nightcrawler2164** [reply]
Here's the steps I followed with a couple minor updates as well which I added as comments to this post, check it out. It's been working great for a couple weeks now with no issues

https://github.com/itsToggle/plex_debrid/issues/492
